docs/MAINTAINERS.md

```diff
@@ -14,41 +14,38 @@ While our maintainers guide Snap, the plugins, and the community forward in a po
 ## Snap Maintainers
 Below is the list of Snap Maintainers, they are largely responsible for approving pull requests and thus have the final say on coding style and implementation. 
 
-| Name              | Tag                |
-|:------------------|:-------------------|
-| Andrzej Kuriata   | @andrzej-k         |  
-| Cody Roseborough  | @IRCody            |  
-| Emily Gu          | @candysmurf        |  
-| Izabella Raulin   | @IzabellaRaulin    |  
-| Joel Cooklin      | @jcooklin          |  
-| Katarzyna Kujawa  | @katarzyna-z       |  
-| Kelly Lyon        | @kjlyon            |  
-| Lukasz Mroz       | @lmroz             |  
-| Marcin Krolik     | @marcin-krolik     |  
-| Marcin Olszewski  | @marcintao         |  
-| Marcin Spoczynski | @sandlbn           |  
-| Maria Rolbiecka   | @mkuculyma         |  
-| Mateusz Kleina    | @mkleina           |  
-| Matt Brender      | @mjbrender         |  
-| Nicholas Weaver   | @lynxbat           |  
-| Olivier Cano      | @kindermoumoute    |  
-| Patryk Matyjasek  | @PatrykMatyjasek   |
-| Rashmi Gottipati  | @rashmigottipati   |  
+| Name              | Tag              |
+|:------------------|:-----------------|
+| Andrzej Kuriata   | @andrzej-k       |
+| Emily Gu          | @candysmurf      |
+| Izabella Raulin   | @IzabellaRaulin  |
+| Joel Cooklin      | @jcooklin        |
+| Katarzyna Kujawa  | @katarzyna-z     |
+| Kelly Lyon        | @kjlyon          |
+| Lukasz Mroz       | @lmroz           |
+| Marcin Krolik     | @marcin-krolik   |
+| Marcin Olszewski  | @marcintao       |
+| Marcin Spoczynski | @sandlbn         |
+| Maria Rolbiecka   | @mkuculyma       |
+| Mateusz Kleina    | @mkleina         |
+| Matt Broberg      | @mbbroberg       |
+| Patryk Matyjasek  | @PatrykMatyjasek |
+| Rashmi Gottipati  | @rashmigottipati |
 
 ### Responsibilities of Snap Maintainers
 Maintainers are expected to contribute regularly in at least one of the following capacities:
 * Direction/Design: Meaningfully contributing to RFC or design comments
 * Code Contributions: Submitting pull requests for either bug fixes or features
 * Support: Active participation in a support role in some form (code reviews, addressing questions, triaging issues)
 
-If a maintainer is consistently unable to perform the above actions they will be asked to withdraw from the maintainer group. This does not preclude them from becoming a maintainer at a later date.
+Maintainer participation is measured by activity visible on their overview page ([example](https://github.com/mbbroberg#overview)) and in our [public Slack](http://slack.snap-telemetry.io). If a maintainer is consistently unable to perform the above actions they will be asked to withdraw from the maintainer group. This does not preclude them from becoming a maintainer at a later date.
```

Comment:
Small suggestion of changing
`Maintainer participation is measured by activity visible on their overview page`
to
`Maintainer participation is measured by activity visible on their Github overview page`

I am also not fully convinced of adding a link to sb overview profile - is it really necessary?